README.md

```diff
@@ -83,7 +83,8 @@ In order to login to WordPress.com with the app you need to create an account ov
 
 After you create an account you can create an application on the [WordPress.com applications manager](https://developer.wordpress.com/apps/).
 
-When creating your application, select "Native client" for the application type. The applications manager currently requires a "redirect URL", but this isn't used for mobile apps. Just use "https://localhost".
+When creating your application, you should select "Native client" for the application type.
+The "**Website URL**", "**Redirect URLs**", and "**Javascript Origins**" fields are required but not used for the mobile apps. Just use "**[https://localhost](https://localhost)**".
```

Comment:
Nitpick: Does the localhost suggestion need to be a link? 

```suggestion
The "**Website URL**", "**Redirect URLs**", and "**Javascript Origins**" fields are required but not used for the mobile apps. Just use "**`https://localhost`**".
```

It's fine either way. Totally up to you 🤷‍♂️

Reply:
Good point! No need to cause unnecessary clicks. 🙂
Fixed → a515cefad63c434d3e6a3be29c9a9cda13853eb1